### ISSUE TITLE
Add the release process

### ISSUE THREAD

**Issue by BewareMyPower** [apache/pulsar-client-cpp#123]

**Comment by tisonkun** [comment]
I read we have https://github.com/apache/pulsar-client-cpp/wiki/Committer-Release-Guide now.

**Comment by shibd** [comment]
> I read we have https://github.com/apache/pulsar-client-cpp/wiki/Committer-Release-Guide now.

I think we should maintain it in the repository. like node js client: https://github.com/apache/pulsar-client-node/pull/243

BTW: The release process may be different between different major versions, and maintaining the process in the repository keeps the process accurate for each branch.

**Comment by tisonkun** [comment]
@shibd Yes. My concern can be added one here: we should still think of 2.8~2.11 until their end of life.

**Comment by shibd** [comment]
> @shibd Yes. My concern can be added one here: we should still think of 2.8~2.11 until their end of life.

It looks like the release process in the wiki matches the latest code. Where is the 2.8~2.10 release process? @BewareMyPower
